emails/templates/emails/wrapped_email.html

```diff
@@ -45,6 +45,11 @@
                   {% bold_violet_text display_email as bolded_email %}
                   {% bold_violet_link SITE_ORIGIN 'Firefox Private Relay' as linked_origin %}
                   {% ftlmsg 'forwarded-email-header-from' display_email=bolded_email linked_origin=linked_origin as email_wrap_forwarded_from %}
+                  {% if recipient_profile.has_premium %}
```

Comment:
Question: is `recipient_profile` the same as regular `profile`s? In other words, can I access this property on there? (/cc @groovecoder)

Reply:
Confirmed that it is!